composer.json

```diff
@@ -24,6 +24,7 @@
         "symfony/http-client": "^5.4",
         "symfony/process": "^5.4",
         "twig/twig": "^2.12",
+        "vimeo/psalm": "^6.0",
```

Comment:
Why is this promoted as a runtime dependency?

Reply:
No longer, thanks